### PR TITLE
Added incentives descriptions + info

### PR DESCRIPTION
**Pull request by garthtravers**

### CODE REVIEW

File: content/incentives.md [hunk 1]
```diff
@@ -12,9 +12,57 @@ Exchange fees are generated whenever a user exchanges one synthetic asset (Synth
...
+
+Duration: 03/13/2020 onwards
+
+Aim: This incentive aims to provide an on-ramp into trading on [Synthetix.Exchange](https://synthetix.exchange) for traders with stablecoins. 
+
+### sETH/ETH liquidity on Uniswap v1
+
+Amount: 4000 SNX per week
+
+Duration: 07/12/2020 onwards
```
Comment (justinjmoses) [inline content/incentives.md:31]: [np] For clarification I'd just use `Jul 12, 220`

Comment (garthtravers) [reply content/incentives.md]: Sure, just saw that's what you've been using in Releases.

File: content/incentives.md [hunk 2]
```diff
@@ -12,9 +12,101 @@ Exchange fees are generated whenever a user exchanges one synthetic asset (Synth
 
 SNX staking rewards are generated through the inflationary monetary policy implemented in March 2018. At the same time as they claim their exchange fees stakers can claim their staking rewards, which are escrowed for a year. The SNX staking rewards enabled by the inflationary supply are set to decrease gradually until September 2023, where it will become a 2.5% annual terminal inflation rate.
 
-### Liquidity Rewards
+## Secondary incentives
 
-5% of the aforementioned inflationary SNX supply is distributed to people providing sETH/ETH liquidity on Uniswap. This is to ensure there is a clear way for traders to enter the system and start trading for the first time (if they hold ETH).
+In addition to the two primary forms of incentives, Synthetix also runs various incentives to create on-ramps, balance out particular dynamics, or trial different behaviour. These are either paid with a portion of the weekly inflationary SNX when implemented at the protocol level, or with SNX from the synthetixDAO treasury if they are just trials.
```
[the remark this answers is not in the document]
Comment (garthtravers) [reply content/incentives.md:17]: I actually disagree Clints — this is a general overview paragraph. I go into detail for each actual incentive, though I could probably go into more detail there I guess.

File: content/incentives.md [hunk 3]
```diff
@@ -12,9 +12,101 @@ Exchange fees are generated whenever a user exchanges one synthetic asset (Synth
...
+
+Staking Contract: [`StakingRewardssUSDCurve`](https://contracts.synthetix.io/StakingRewardssUSDCurve)
+
+Source: [`Curvepool` (via `Unipool`)](https://github.com/Synthetixio/Unipool/blob/curverewards/contracts/Unipool.sol)
+
+Aim: This incentive aims to provide an on-ramp into trading on [Synthetix.Exchange](https://synthetix.exchange) for traders with stablecoins.
+
+### sETH/ETH liquidity on Uniswap v1
+
+Amount: `4,000 SNX` per week
```
[the remark this answers is not in the document]
Comment (garthtravers) [reply content/incentives.md:35]: I added a deprecated incentives section, good idea

File: content/incentives.md [hunk 4]
```diff
@@ -12,9 +12,101 @@ Exchange fees are generated whenever a user exchanges one synthetic asset (Synth
...
+
+Staking Contract: [`StakingRewardssETHUniswapV1`](https://contracts.synthetix.io/StakingRewardssETHUniswapV1)
+
+Source: [`Unipool`](https://github.com/Synthetixio/Unipool/blob/master/contracts/Unipool.sol)
+
+Aim: This incentive aims to provide an on-ramp into trading on [Synthetix.Exchange](https://synthetix.exchange) for traders with ETH.
+
+### iETH holders (trial)
+
+Amount: `32,000 SNX` per week
```
[the remark this answers is not in the document]
Comment (garthtravers) [reply content/incentives.md:49]: We can update it when it gets implemented

File: content/incentives.md [hunk 5]
```diff
@@ -12,9 +12,101 @@ Exchange fees are generated whenever a user exchanges one synthetic asset (Synth
...
 
-### Liquidity Rewards
+## Secondary incentives
 
-5% of the aforementioned inflationary SNX supply is distributed to people providing sETH/ETH liquidity on Uniswap. This is to ensure there is a clear way for traders to enter the system and start trading for the first time (if they hold ETH).
+In addition to the two primary forms of incentives, Synthetix also runs various incentives to create on-ramps, balance out particular dynamics, or trial different behaviour. These are either paid with a portion of the weekly inflationary SNX when implemented at the protocol level, or with SNX from the synthetixDAO treasury if they are just trials.
+
+### sUSD/DAI-USDC-USDT liquidity on Curve
+
+Amount: `48,000 SNX` per week
```
[the remark this answers is not in the document]
Comment (garthtravers) [reply content/incentives.md:21]: Not implemented yet

**Pull request merged**